src/commands/leaderboard.ts

```diff
@@ -0,0 +1,36 @@
+// Dependencies
+import { Telegraf, ContextMessageUpdate } from 'telegraf'
+import { User } from '../models/user';
+import { getLeaderBalance } from '../models'
+import { getName } from '../helpers/name'
+
+interface UserForLeaderBoard {
+    username: String,
+    balance: Number,
+}
+
+const getUser = async (ctx: any, user: User): Promise<UserForLeaderBoard> => {
```

Comment:
1. Remove return time (it will be inferred)
2. `ctx` should be `ContextMessageUpdate`